### Laelaps laelaps_control/WriteGpio service
# 
# Write GPIO pin value(s).
#

Gpio[] gpio     # GPIO pin state(s) to write
---
uint8[] state   # pin states(s) after write
